backend/service/k8s/k8s_test.go

```diff
@@ -134,9 +132,9 @@ func TestInterceptError(t *testing.T) {
 }
 
 func TestGetK8sClientset(t *testing.T) {
-	tempfile, _ := ioutil.TempFile("", "")
+	tempfile, _ := os.CreateTemp("", "")
 	defer os.Remove(tempfile.Name())
-	_ = ioutil.WriteFile(tempfile.Name(), []byte(testConfig), 0500)
+	_ = os.WriteFile(tempfile.Name(), []byte(testConfig), 0o500)
```

Comment:
Just for my understanding why `s/0500/0o500/`

Reply:
I was gonna say `gofmt` does it, but looks like it doesn't yet (https://github.com/golang/go/issues/33363). So it was probably `gofumpt` running in my editor. Let me revert it.